### PR TITLE
Fixes for refseq data processing

### PR DESCRIPTION
**Pull request by khajoue2**
* Updated modify_gtf script to add gene_name attribute
* Updated Tagsort to parse gene_id and gene_name fields only and print more helpful errors.

### CODE REVIEW

**Review by kevinpalis — APPROVED** [review]
Looks good to me. Thanks, Farzaneh!

File: tools/docker_versions.tsv [hunk 1]
```diff
@@ -16,3 +16,4 @@ us.gcr.io/broad-gotc-prod/warp-tools:1.0.1-1686671920
 us.gcr.io/broad-gotc-prod/warp-tools:1.0.5-1692706846
 us.gcr.io/broad-gotc-prod/warp-tools:1.0.6-1692962087
 us.gcr.io/broad-gotc-prod/warp-tools:1.0.7-1695393479
+us.gcr.io/broad-gotc-prod/warp-tools:1.0.6-1698879268
```
Comment (jessicaway) [inline tools/docker_versions.tsv]: We should build a new version. Here we are producing version 1.0.6 after 1.0.7

**Review by jessicaway — APPROVED** [review]
This looks good to me, although I will admit I don't really understand how the logic changed.

File: tools/docker_versions.tsv (hunk 1 above)
Comment (ekiernan) [reply tools/docker_versions.tsv]: I think it will be 1.0.9

Comment (jessicaway) [reply tools/docker_versions.tsv]: When we build in the github actions I don't believe this gets updated anyways

**Pull request merged**